src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
Kindly import the categoryReducer and configure it in the store. This would ensure it is tracked by your store as well.